website/templates/emails/add_sso_email_osf4i.html.mako

```diff
@@ -0,0 +1,18 @@
+<%inherit file="notify_base.mako" />
+
+<%def name="content()">
+<tr>
+  <td style="border-collapse: collapse;">
+
+    Hello ${user.fullname},<br>
+    <br>
+    Thank you for connecting to OSF through your institution. This email address &lt;${email_to_add}>&gt; has been added to your account as an alternate email address. If you would like to make this your primary contact address you can find instructions to do so at: <a href="https://help.osf.io/article/238-add-a-new-email-address-to-your-account#change-your-primary-email-address-on-your-account">https://help.osf.io/article/238-add-a-new-email-address-to-your-account#change-your-primary-email-address-on-your-account</a>.
```

Comment:
@cslzchen fix this

> `&lt;${email_to_add}>&gt;`

Reply:
In addition, this link looks too long and maybe "Change Your Primary Email Address On Your Account"